h/assets.ini

```diff
@@ -24,3 +24,9 @@ site_css =
 help_page_css =
   styles/icomoon.css
   styles/help-page.css
+
+create_group_form_css =
+  styles/create-group-form.css
+
+create_group_form_js =
+  scripts/create-group-form.bundle.js
```

Comment:
I expect we'll want to make this a bit more general in future so we have a bundle for all the new UI rather than just the "create group" UI. Only once that gets large enough would we then need to split it up again.